core/model/src/main/java/org/eclipse/sw360/antenna/api/IEvaluationResult.java

```diff
@@ -49,4 +50,15 @@ public static Severity fromValue(String v) {
      * @return All artifacts that have broken this rule.
      */
     Set<Artifact> getFailedArtifacts();
+
+    default String resultAsMessage() {
+        return String.format("%s (%s): %s%n          %s",
+                getId(),
+                getSeverity(),
+                getFailedArtifacts()
```

Comment:
Sorry to be picky, but I checked the code and we normally do it like this:
```java
variable.potentialfunction().stream()
    .map(...)
    .collect(...)
```
Though I just checked the code and saw that there is a big inconsistency in formatting... We should maybe set a standard now and then adhere to it.